x-pack/plugins/infra/server/lib/alerting/inventory_metric_threshold/inventory_metric_threshold_executor.ts

```diff
@@ -200,6 +232,15 @@ const buildReasonWithVerboseMetricName = (
   return buildReason(resultWithVerboseMetricName);
 };
 
+const buildAlertData = (inventoryItem: any, nextState: any): AlertData => {
```

Comment:
Is there a way we can use the correct types instead of `any`?

```suggestion
const buildAlertData = (inventoryItem: ConditionResult, nextState: AlertState): AlertData => {
```

Reply:
Not needed anymore